src/main/java/frc/robot/subsystems/intake/Command/Feed.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.intake.Command;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.intake.Intake;
+
+public class Feed extends CommandBase {
+    private Intake intake = Intake.getINSTANCE();
+    private double power;
+
+    public Feed(Intake intake, double power) {
+        addRequirements(intake);
```

Comment:
Well, you don't use the parameters you asked...

Reply:
done.